File: 2240_Number_of_Ways_to_Buy_Pens_and_Pencils/main.go
package main

func waysToBuyPensPencils(total int, cost1 int, cost2 int) int64 {
	var ans int64
	var expensive int
	var cheap int
	if cost1 > cost2 {
		expensive = cost1
		cheap = cost2
	} else {
		expensive = cost2
		cheap = cost1
	}

	for total >= 0 {
		ans += int64(total/cheap) + 1
		total -= expensive
	}

	return ans
}

File: 2240_Number_of_Ways_to_Buy_Pens_and_Pencils/main.py
class Solution:
    def waysToBuyPensPencils(self, total: int, cost1: int, cost2: int) -> int:
        ans = 0
        expensive = max(cost1, cost2)
        cheap = min()
        if cost1 > cost2:
            expensive = cost1
            cheap = cost2
        else:
            expensive = cost2
            cheap = cost1

        while total >= 0:
            ans += total // cheap + 1
            total -= expensive
        return ans
